### PR TITLE
Remove useless pom imports.

### PR DESCRIPTION
**Pull request by gautierbureau**
Signed-off-by: Gautier Bureau <gautier.bureau@rte-france.com>

### CODE REVIEW

**Review by geofjamg — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -43,22 +43,6 @@
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>com.google.cloud.tools</groupId>
```
Comment (gautierbureau) [inline pom.xml:47]: important to have mvn clean install -Dpowsybl.docker.install

File: pom.xml [hunk 2]
```diff
@@ -43,22 +43,6 @@
...
-                <artifactId>jib-maven-plugin</artifactId>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-failsafe-plugin</artifactId>
-            </plugin>
-            <plugin>
-                <groupId>org.springframework.boot</groupId>
```
Comment (gautierbureau) [inline pom.xml:55]: important to have mvn package produce package.jar

File: pom.xml [hunk 3]
```diff
@@ -43,22 +43,6 @@
...
-                <artifactId>spring-boot-maven-plugin</artifactId>
-            </plugin>
-            <plugin>
-                <groupId>pl.project13.maven</groupId>
```
Comment (gautierbureau) [inline pom.xml:59]: important to have the git commit in the actuator info

File: pom.xml [hunk 4]
```diff
@@ -198,23 +158,5 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.junit.vintage</groupId>
```
Comment (gautierbureau) [inline pom.xml:202]: important to have tests being actually launched

File: pom.xml [hunk 5]
```diff
@@ -83,17 +78,6 @@
...
                 <scope>import</scope>
             </dependency>
-            <!-- project specific dependencies -->
-            <dependency>
-                <groupId>org.antlr</groupId>
-                <artifactId>ST4</artifactId>
-                <version>${string-template.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.apache.commons</groupId>
```
Comment (gautierbureau) [inline pom.xml:93]: important to have swagger working

File: pom.xml [hunk 6]
```diff
@@ -83,17 +78,6 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
-            <!-- project specific dependencies -->
-            <dependency>
-                <groupId>org.antlr</groupId>
```
Comment (gautierbureau) [inline pom.xml:88]: dependencie got from network-store by transitivity but should be kept

**Review by jonenst — APPROVED** [review]

**Pull request merged**